test/linalg/matmul.jl

```diff
@@ -59,6 +59,7 @@ A = rand(1:20, 5, 5) .- 10
 B = rand(1:20, 5, 5) .- 10
 @test At_mul_B(A, B) == A'*B
 @test A_mul_Bt(A, B) == A*B'
+@test Ac_mul_Bt(A, B) == A'*B.'
```

Comment:
should this actually get tested on complex inputs so we can be sure the conjugation behavior is right?